pycytominer/cyto_utils/util.py

```diff
@@ -245,9 +245,16 @@ def check_image_features(image_features, image_columns):
         Nothing is returned.
     """
 
+    if "Image" in list(set(_.split("_")[0] for _ in image_columns)):
```

Comment:
```suggestion
    if "Image" in list(set(img_col.split("_")[0] for img_col in image_columns)):
```

It's just easier to read this way :)

Reply:
(I realize this was how it was originally, introducing the change here and below)